client/client.go

```diff
@@ -205,28 +205,37 @@ func (c *Client) useInsecureHTTPClient(insecure bool) *http.Transport {
 	return transport
 }
 
-func (c *Client) MakeRestRequest(method string, path string, body *container.Container, authenticated bool) (*http.Request, error) {
+func (c *Client) MakeFullUrl(method string, path string) (string, error) {
 	if c.platform == "nd" && path != "/login" {
-		if strings.HasPrefix(path, "/") {
-			path = path[1:]
-		}
-		path = fmt.Sprintf("mso/%v", path)
+		path = strings.TrimPrefix(path, "/")
```

Comment:
Should we considering adding some logging that trimming is one?
Should we considering doing a regex check on the end result, to validate a accepted url is returned and else log some additional warning/error? This to avoid that weird error message being displayed in terraform output.

Reply:
We already log the resulting url in debug would this be enough? The url.Parse method handles validation of url strings and will display an error if it was invalid. Url parsing is quite complex and we don't want to restrict it too much in case we block a url that could be valid.